nixos/modules/services/backup/borgbackup.nix

```diff
@@ -243,6 +255,16 @@ in {
             example = "/home/user";
           };
 
+          backupCommand = mkOption {
+            type = types.str;
+            description = ''
+              Backup the stdout of this program instead of filesystem paths.
+              You must set <option>paths</option> to <literal>-</literal>
+              to use this option.
+            '';
+            example = "/path/to/createZFSsend.sh";
+          };
+
```

Comment:
Setting `cfg.backupCommand` only makes sense in combination with `cfg.paths == ["-"
].`. It would be nice to add an assertion to check that.

Reply:
I'll try to do that this weekend

Reply:
I've added the assertion in a new commit